### PR TITLE
[v7r2] Various fixes

### PR DESCRIPTION
**Pull request by fstagni**
BEGINRELEASENOTES

*Monitoring
FIX: Adapting RMSMonitoring type to ES7

ENDRELEASENOTES

Plus adding a couple integration tests, after moving them to pytest format.

### CODE REVIEW

**Comment by atsareg** [comment]
Close/reopen to restart tests

**Comment by fstagni** [comment]
I will merge this after Chris's PR on python3 is merged (rebasing).

**Comment by fstagni** [comment]
Rebase done.

**Review by atsareg — APPROVED** [review]

**Pull request merged**
